tests/robustness/traffic.go

```diff
@@ -64,26 +64,39 @@ func simulateTraffic(ctx context.Context, t *testing.T, lg *zap.Logger, clus *e2
 	limiter := rate.NewLimiter(rate.Limit(config.maximalQPS), 200)
 
 	startTime := time.Now()
+	c, err := NewClient(endpoints, ids, startTime)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer c.Close()
 	wg := sync.WaitGroup{}
 	for i := 0; i < config.clientCount; i++ {
 		wg.Add(1)
-		endpoints := []string{endpoints[i%len(endpoints)]}
-		c, err := NewClient(endpoints, ids, startTime)
+		c, err := NewClient([]string{endpoints[i%len(endpoints)]}, ids, startTime)
 		if err != nil {
 			t.Fatal(err)
 		}
 		go func(c *recordingClient, clientId int) {
 			defer wg.Done()
 			defer c.Close()
 
-			config.traffic.Run(ctx, clientId, c, limiter, ids, lm)
+			config.traffic.Run(ctx, clientId, c, limiter, ids, lm, finish)
 			mux.Lock()
 			h = h.Merge(c.history.History)
 			mux.Unlock()
 		}(c, i)
 	}
 	wg.Wait()
 	endTime := time.Now()
+
+	// Ensure that last operation is succeeds
+	time.Sleep(time.Second)
```

Comment:
We already have `wg.Wait()` to ensure traffic.run to finish. Is there any reason to wait 1 second here?

Reply:
Same us before, we require some time without traffic to get periodic watch notification.

Reply:
Sounds good to me. When the last request is timeout and the server already received the request, it might need time to handle it. Thanks for the comment.